pkg/member/member_control.go

```diff
@@ -130,10 +130,10 @@ func (m *memberControl) AddMemberAsLearner(ctx context.Context) error {
 	memAddCtx, cancel := context.WithTimeout(ctx, EtcdTimeout)
 	defer cancel()
 	start := time.Now()
-	response, err := cli.MemberAddAsLearner(memAddCtx, []string{memberURL})
+	response, err := cli.MemberAddAsLearner(memAddCtx, memberPeerURLs)
 	if err != nil {
 		if errors.Is(err, rpctypes.Error(rpctypes.ErrGRPCPeerURLExist)) || errors.Is(err, rpctypes.Error(rpctypes.ErrGRPCMemberExist)) {
-			m.logger.Infof("Member %s already part of etcd cluster", memberURL)
+			m.logger.Infof("Member %s already part of etcd cluster", m.podName)
```

Comment:
log should also contains the member URL as `rpctypes.Error(rpctypes.ErrGRPCPeerURLExist)` refering to URL

Reply:
I only changed it because the code is using podname in the next few lines [here](https://github.com/gardener/etcd-backup-restore/blob/0bcd93ae1f3f60acce9733df9363d76c833c0568/pkg/member/member_control.go#L138-L141)

Do you want to return podname and urls ( multiple urls is possible in few cases )  both?